### PR TITLE
adding hanan hadi zainb

### PR DESCRIPTION
**Pull request by aliabdullah24**
please check and let me know in case of any issue

### CODE REVIEW

File: Hungry 2.txt [hunk 1]
```diff
@@ -4,3 +4,8 @@ hogen
 cena
 pida
 riyan
+hadi 
+hani 
+zaron
+haniya
+zainab
```
Comment (CodeCraft22) [inline Hungry 2.txt:11]: also add fahad in it

**Comment by CodeCraft22** [comment]
please add fahad in it

File: Hungry 2.txt (hunk 1 above)
Comment (CodeCraft22) [inline Hungry 2.txt:11]: add 125

Comment (aliabdullah24) [reply Hungry 2.txt:11]: done my yaki parts

Comment (CodeCraft22) [reply Hungry 2.txt:11]: good

**Pull request merged**